server/routes/products.js

```diff
@@ -0,0 +1,73 @@
+const db = require('APP/db/models');
+const Product = db.Product;
+
+const Router = require('express').Router();
+
+
+//GET all the Products
+Router.get('/', (req, res, next) => {
+	Product.findAll({})
+	.then((products) => {
+		res.json(products);
+	})
+	.catch(next);
+});
+
+//GET single Product
+Router.get('/:productId', (req, res, next) => {
+	Product.findById(
+		req.params.productId
+	)
+	.then((singleProduct) =>{
+		res.json(singleProduct);
+	})
+	.catch(next);
+});
+
+//DELETE product
+Router.delete('/:productId', (req, res, next) => {
+	Product.destroy({
+		where: {
+			id: req.params.productId
+		}
+	})
+	.then((numDestroyedRows) => {
+		res.status(200).json(numDestroyedRows);
+	})
+	.catch(next);
+});
+
+//PUT (update) a product
+Router.put('/:productId', (req, res, next) => {
+	Product.update({ name: req.body.name,
```

Comment:
For the first argument in update, we might want to replace with req.body even though we discussed that there might be security issues. Omri pointed out to me and Surabhi that one benefit of doing just req.body is that we won't have to go through each of the routes to update specific fields if we decided to add or delete any later. He said we could then set up security measures elsewhere although I'm not sure how exactly.